server/metrics/metrics.go

```diff
@@ -3191,6 +3191,27 @@ var (
 		GroupID,
 		StatusLabel,
 	})
+
+	// ## Cache Proxy Remote Hit-Tracker Metrics
```

Comment:
nit: no need to introduce a new docs section IMO, since there is already a "Cache proxy" section in the prometheus metrics docs. Can also remove the atime-specific section

Reply:
Done